docs/source/updating_capabilities.md

```diff
@@ -0,0 +1,341 @@
+
+### `Channel` group
+
+Navigate to [Step 1: Pull and translate the config](./config_update.html#step-1-pull-and-translate-the-config).
+Once you have a `modified_config.json`, add the capabilities to the `Channel`
+group of the config (as listed in `capabilities.json`) using this command:
+
+```
+jq -s '.[0] * {"channel_group":{"values": {"Capabilities": .[1].channel}}}' config.json ./capabilities.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#step-3-re-encode-and-submit-the-config).
+
+Note that the `mod_policy` for this capability defaults to requiring signatures
+from both the `MAJORITY` of `Admins` in the `Application` and `Orderer` groups.
+In other words, both a majority of the peer organization admins and ordering
+service organization admins must sign this request.
+
+### `Application` group
+
+Navigate to [Step 1: Pull and translate the config](./config_update.html#step-1-pull-and-translate-the-config).
+Once you have a `modified_config.json`, add the capabilities to the `Application`
+group of the config (as listed in `capabilities.json`) using this command:
+
+```
+jq -s '.[0] * {"channel_group":{"groups":{"Application": {"values": {"Capabilities": .[1].application}}}}}' config.json ./capabilities.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#step-3-re-encode-and-submit-the-config).
+
+Note that the `mod_policy` for this capability defaults to requiring signatures
+from the `MAJORITY` of `Admins` in the `Application` group. In other words, a
+majority of peer organizations will need to approve. Ordering service admins
+have no say in this capability.
+
+**As a result, be very careful to not change this capability to a level that does
+not exist**. Because ordering nodes neither understand nor validate `Application`
+capabilities, they will approve a configuration to any level and send the new
+config block to the peers to be committed to their ledgers. However, the peers
+will be unable to process the capability and will crash. And even it was possible
+to drive a corrected configuration change to a valid capability level, the
+previous config block with the faulty capability would still exist on the ledger
+and cause peers to crash when trying to process it.
+
+This is one reason why a file like `capabilities.json` can be useful. It prevents
+a simple user error --- for example, setting the `Application` capability to `V20`
+when the intent was to set it to `V2_0` --- that can cause a channel to be
+unusable and unrecoverable.
+
+## Verify a transaction after capabilities have been enabled
+
+It's a best practice to ensure that capabilities have been enabled successfully
+with a chaincode invoke on all channels. If any nodes that do not understand new
+capabilities have not been upgraded to a sufficient binary level, they will crash.
```

Comment:
The crash would happen when a peer receives the block with the config update, not when executing chaincode.